examples/metrics/grafana-dashboards/strimzi-kafka-connect.json

```diff
@@ -124,7 +124,7 @@
       "tableColumn": "",
       "targets": [
         {
-          "expr": "sum(kafka_connect_worker_connector_count{strimzi_io_kind=~\"KafkaConnect.*\",strimzi_io_cluster=\"$strimzi_connect_cluster_name\"})",
+          "expr": "sum(kafka_connect_worker_connector_count{strimzi_io_kind=~\"KafkaConnect.*\",strimzi_io_cluster=\"$strimzi_connect_cluster_name\", strimzi_io_kind=~\"KafkaConnect.*\"})",
```

Comment:
It is already on the beginning ... does it make sense to have it twice?

Reply:
Ah, no. I am just blind.